src/windows/staff.c

```diff
@@ -1325,6 +1325,17 @@ void window_staff_viewport_init(rct_window* w){
 	window_invalidate(w);
 }
 
+void staff_get_valid_skins(int* ebx)
+{
+	init_scenery();
+	*ebx = 0;
```

Comment:
Resharper did it that way. But if you prefer it, I can also change it. But at all there is no real difference.